src/Microsoft.Data.SqlClient/netcore/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+using System.Runtime.CompilerServices;
```

Comment:
`using System.Runtime.CompilerServices;` is the only directive that is actually needed, the others seem redundant.